src/components/App.js

```diff
@@ -1,14 +1,92 @@
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
+import calculate from '../logic/calculate';
 
-const App = () => {
-  return (
-    <div className="CalculatorApp">
-      <Display />
-      <ButtonPanel />
-    </div>
-  );
-};
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      next: null,
+      operation: null,
+      total: null,
+    };
+  }
+
+  handleNumber(buttonName) {
+    let { total, next } = this.state;
+    const { operation } = this.state;
+    if (operation === null) {
+      if (total === null) {
+        total = buttonName;
+      } else {
+        total += buttonName;
+      }
+    } else if (next === null) {
+      next = buttonName;
+    } else {
+      next += buttonName;
+    }
+    return { next, operation, total };
+  }
```

Comment:
There's a `setState` method to change the value of the state. Please let's put all the calculation logic in the calculate module